client/src/containers/editHours/EditHours.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from "react";
+import getDataFromHour from "../../services/axios_sev/getDataFromHour";
+import { connect, useDispatch } from "react-redux";
+import { months,filterData } from "../../services/editHoursSev";
+import {monthHours} from "../../reducers/actions/index"
+import UserHours from "./UserHours";
+
+function EditHours({ userToken, userId }) {
+  const [data, setData] = useState([]);
+  const [monthData, setMonthData] = useState([]);
+  const [workThisMonth, setWorkThisMonth] = useState(false);
+  const dispatch = useDispatch();
+  //need to figure out how to call when the component also updates
+  useEffect(() => {
+    getDataFromHour(userToken, userId).then((res) => {
+      setData(res.data);
+    });
+  }, []);
+
+const handleId= async(e)=>{
+  const id = await e.target.id;
+  filterMonth(id)
+}
+
+  
+  const filterMonth =  (id) => {
+      filterData(data,id).then(response=>
+     { setMonthData(response)
```

Comment:
@software-bastards @reMRKableDev  I have no experience in react hooks. But if I read this it looks like it's updating local state via hooks(?) but also dispatching a redux action. Feels double to me. Not sure what best practice is in this one :)

Reply:
Indeed. I will just connect the component and pass the props via the store.